cmd/vpn_server/app/firewall.go

```diff
@@ -21,9 +22,10 @@ import (
 
 func firewallCommand() *cobra.Command {
 	var (
-		device        string
-		mode          string
-		shootNetworks []string
+		device         string
+		mode           string
+		shootNetworks  []string
+		seedPodNetwork string
```

Comment:
Let's make the name more explicit. It is not the seed pod network, but the IPv4 seed pod network, which is hopefully only a single network. In other words, please at least adapt the name.
(As discussed above, we could think of multiple IPv4 pod networks, but this is probably not relevant today.)